speculation-rules/prerender/restriction-midi-sysex.https.html

```diff
@@ -21,7 +21,7 @@
   // is deferred during prerendering so the test would trivially pass without
   // the permission.
   await test_driver.set_permission(
-      {name: 'midi', sysex: true}, 'granted', true);
```

Comment:
The speculation-rules/ test all seem to set `one_realm` to true; my question about whether this will cause any regressions in WebKit also applies.

Reply:
Nope. It shouldn’t